net/csshnpd/README.md

```diff
@@ -0,0 +1,134 @@
+# Package: net/csshnpd
+
+This package contains the
+[C NoPorts daemon](https://github.com/atsign-foundation/noports/tree/trunk/packages/c/sshnpd)
+
+NoPorts can be used to connect to a device using TCP protocols such as SSH
+without needing an open port exposed to the Internet.
+
+Before installing NoPorts on OpenWrt it's recommended that you install the
+NoPorts client and activate client and device keys on the machine you're
+going to connect from.
+
+The [installation overview](https://docs.noports.com/installation) gives
+a high level run through of the process, and specific guides for Linux,
+MacOS and Windows can be found in the navigation bar of that page.
+
+## Installing csshnpd
+
+### LuCI installation
+
+Browse to System > Software then type `cssh` into the Filter: box.
+
+Then press the `Install...` button beside the csshnpd package.
+
+It is recommended that you also install the `luci-app-csshnpd` package
+to enable configuration in LuCI.
+
+### CLI installation
+
+To install on snapshot OpenWrt:
+
+```sh
+apk update
+apk add csshnpd
+```
+
+For older versions please refer to the
+[OpenWrt Installation Guide](https://docs.noports.com/installation/openwrt-installation-guide)
+
+## Configuring the sshnpd service
```

Comment:
Do we need to have this readme in our repository? Shouldnt it be rather some doku page e.g. on OpenWrt wiki?

Reply:
README removed in https://github.com/openwrt/packages/pull/26518/commits/f1ab269d6fdbff238c2a5b3c90a7293e6db44d15 and I'd be happy to create a wiki entry once this is merged.